ci_framework/roles/edpm_build_images/tasks/run.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Set DIB elements path
+  ansible.builtin.set_fact:
+    cifmw_elements: "{{ cifmw_edpm_build_images_elements | join(':') }}"
```

Comment:
You can directly pass "{{ cifmw_edpm_build_images_elements | join(':') }}" to     ELEMENTS_PATH , no need to create a new variable


    ELEMENTS_PATH: "{{ cifmw_edpm_build_images_elements | join(':') }}